packages/sdk/connect/src/connection.ts

```diff
@@ -348,12 +348,12 @@ export class Connection {
         })
     }
     const [maxFeePerGas, maxPriorityFeePerGas] = await Promise.all(calls)
-    return {
-      ...tx,
-      gasPrice: undefined,
-      maxFeePerGas,
-      maxPriorityFeePerGas,
-    }
+
+    tx.maxFeePerGas = maxFeePerGas
+    tx.maxPriorityFeePerGas = maxPriorityFeePerGas
+
+    delete tx.gasPrice
+    return tx
```

Comment:
I guess I could revert this as it's a leftover from when I was estimating `maxFeeInFeeCurrency` here